coverage/.last_run.json

```diff
@@ -1,5 +1,5 @@
 {
   "result": {
-    "covered_percent": 98.6
+    "covered_percent": 56.89
```

Comment:
What happened here??

Reply:
Ran solely integration tests, not whole suite. Can disregard.